src/lib/components/sections/Wallets.svelte

```diff
@@ -21,11 +21,11 @@
 			img: 'kalium-wallet'
 		},
 		{
-			title: 'BananoVault',
+			title: 'TheBananoStand',
 			description: 'Browser-based lightwallet that also supports Ledger Nano S. Ideal for desktop.',
-			buttonText: 'Visit BananoVault',
-			url: bananoWallets.vault.url,
-			img: 'banano-vault'
+			buttonText: 'Visit BananoStand',
```

Comment:
I made this "BananoStand" instead of "TheBananoStand" so the button text wouldn't break onto 2 lines.